Sources/ArcGISToolkit/Components/FeatureFormView/FeatureFormView.swift

```diff
@@ -85,8 +91,9 @@ public struct FeatureFormView: View {
     /// Initializes a form view.
     /// - Parameters:
     ///   - featureForm: The feature form defining the editing experience.
-    public init(featureForm: FeatureForm) {
+    public init(featureForm: FeatureForm, utilityNetwork: UtilityNetwork? = nil) {
```

Comment:
Instead of having the FF constructor expand to encompass new features, we should probably have the `UtilityNetworkAssociationsFormElement` have a `utilityNetwork` property. This would be keeping with the idea that the elements are the "models" for the display. For now, this is fine.

Reply:
100%, this was to get the element functional for the time being. I am definitely looking to avoid initializer changes if they can be avoided.